src/current/_includes/cockroachcloud/create-cluster-api-serverless.md

```diff
@@ -0,0 +1,92 @@
+**API request**
+
+To create a **Serverless** cluster, send a `POST` request to the `/v1/clusters` endpoint, specifying the following parameters:
+
+- `name`: your cluster's name, a short string with no whitespace.
+- `provider`: "GCP" or "AWS".
+- `primary_region`: (optional) Specify which region should be made the primary region. This is only applicable to multi-region Serverless clusters. This field is required if you create the cluster in more than one region.
+- `regions`: An array of strings specifying the cloud provider's zone codes. For example, for Oregon, set region_name to "us-west2" for GCP and "us-west-2" for AWS.
+- `usage_limits`:
+  - `request_unit_limit`: (int64) the maximum number of request units that the cluster can consume during the month. If this limit is exceeded, then the cluster is disabled until the limit is increased, or until the beginning of the next month when more free request units are granted. It is an error for this to be zero.
+  - `storage_mib_limit`: (int64) the maximum number of Mebibytes of storage that the cluster can have at any time during the month. If this limit is exceeded, then the cluster is throttled; only one SQL connection is allowed at a time, with the expectation that it is used to delete data to reduce storage usage. It is an error for this to be zero.
+- `spend_limit`: (integer) the maximum monthly charge for a cluster, in US cents. We recommend using `usage_limits` instead, since `spend_limit` will be deprecated in the future.
+
+[API reference](https://www.cockroachlabs.com/docs/api/cloud/v1.html#post-/api/v1/clusters)
+
+{% include_cached copy-clipboard.html %}
+~~~ shell
+curl --request POST \
+--url 'https://management-staging.crdb.io/api/v1/clusters' \
+--header "Authorization: Bearer $COCKROACH_API_KEY" \
+--data @create-serverless-cluster-create.json
+~~~
+
+{% include_cached copy-clipboard.html %}
+~~~ json
+#create-serverless-cluster-create.json
+{
+  "name": "exemplary-cockroach",
+  "provider": "GCP",
+  "spec": {
+    "serverless": {
+      "regions": [
+        "us-central1"
+      ]
+    }
+  }
+}
+~~~
+
+**API response**
+
+Upon success, the API will return information about the newly created cluster:
+
+Refer to the [API reference](https://www.cockroachlabs.com/docs/api/cloud/v1.html#post-/api/v1/clusters) for full details.
+  - `id`: Your cluster's UUID.
+  - `name`: The name of the cluster you specified when creating the cluster.
+  - `cockroach_version`: The [release version of CockroachDB](../releases/index.html) running on your cluster.
+  - `account_id`: <!-- ??? why is this null when I do it?  -->
```

Comment:
Reminder to fill this in and remove the comment

Reply:
This markdown comment is messing up the preceding bullet points, so you may want to double check the preview after you have removed.